cfme/tests/cloud_infra_common/test_html5_vm_console.py

```diff
@@ -0,0 +1,188 @@
+            "VM Console didn't prompt for Password"
+        # Enter Password:
+        vm_console.send_keys("{}\n".format(console_vm_password))
+
+        time.sleep(5)  # wait for login to complete
+
+        # This regex can find if there is a word 'login','password','incorrect' present in
+        # text, irrespective of its case
+        regex_for_login_password = re.compile(r'\blogin\b | \bpassword\b| \bincorrect\b',
+         flags=re.I | re.X)
+
+        def _try_login():
+            """
+            Try to read what is on present on the last line in console.
+
+            If it is word 'login', enter username, if 'password' enter password, in order
+            to make the login successful
+            """
+            if vm_console.find_text_on_screen(text_to_find='login', current_line=True):
+                vm_console.send_keys(console_vm_username)
+
+            if vm_console.find_text_on_screen(text_to_find='Password', current_line=True):
+                vm_console.send_keys("{}\n".format(console_vm_password))
+            # if the login attempt failed for some reason (happens with RHOS-cirros),
+            # last line of the console will contain one of the following words:
+            # [login, password, incorrect]
+            # if so, regex_for_login_password will find it and result will not be []
+            # .split('\n')[-1] splits the console text on '\n' & picks last item of resulting list
+            result = regex_for_login_password.findall(vm_console.get_screen_text().split('\n')[-1])
+            return result == []
+
+        # if _try_login() returns True, it means we did not find any of words
+        # [login, password, incorrect] on last line of console text, which implies login success
+        wait_for(func=lambda: _try_login(), timeout=300, delay=5)
+
+        logger.info("Wait to get the '$' prompt")
+        if provider.one_of(VMwareProvider):
+            vm_console.wait_for_text(text_to_find="root@photon", timeout=200)
+        else:
+            time.sleep(15)
+
+        # create file on system
+        vm_console.send_keys("touch blather\n")
+        logger.info("Wait 5 sec for file creation")
+        time.sleep(5)  # Wait for file creation
+
+        # Test pressing ctrl-alt-delete...we should be able to get a new login prompt:
+        vm_console.send_ctrl_alt_delete()
+        vm_console.wait_for_text(text_to_find="login:", timeout=200, to_disappear=True)
+        assert vm_console.wait_for_text(text_to_find="login:", timeout=200) is not None,\
+            "VM Console didn't prompt for Login"
+
+        with ssh.SSHClient(hostname=vm_obj.ip_address, username=console_vm_username,
+                password=console_vm_password) as ssh_client:
+            # if file was created in previous steps it will be removed here
+            # we will get instance of SSHResult
+            command_result = ssh_client.run_command("rm bla*", ensure_user=True)
```

Comment:
Thanks for explanation @jamesooden.

@kedark3 Please add comment detailing the failure here and that this is a workaround

**Consider**

Generating a random filename with a prefix, and `rm <prefix>*` here so that it can be unique test-to-test.